scrapyohio/scrapyohio/spiders/warren.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from scrapy import Request
+
+
+class WarrenSpider(scrapy.Spider):
+    name = 'warren'
+    allowed_domains = ['co.warren.oh.us']
+    start_urls = ['http://www.co.warren.oh.us/property_search/summary.aspx?account_nbr=6150660']
+
+    def start_requests(self):
+        # We want to assign headers for each request triggered. Override the request object
+        # sent over to include Lucia's contact information
+        for url in self.start_urls:
+            yield Request(url, dont_filter=True,
+                          headers={
+                            "Info": "Request made as part of an Eye on Ohio Journalism project.",
+                            "Questions": "With questions, contact Lucia Walinchus at lucia@eyeonohio.com."
+                           }
+                          )
```

Comment:
@walinchus FYI: Here's the information getting sent over in the header of each request.